packages/components/src/components/contract-card/contract-card-items/contract-card-header.jsx

```diff
@@ -19,22 +19,40 @@ const ContractCardHeader = ({
     has_progress_slider,
     id,
     is_mobile,
+    is_sold: is_contract_sold,
     is_sell_requested,
     is_valid_to_sell,
     onClickSell,
     server_time,
 }) => {
     const current_tick = contract_info.tick_count ? getCurrentTick(contract_info) : null;
-    const { underlying, multiplier, contract_type, shortcode, purchase_time, date_expiry, tick_count, is_sold } =
-        contract_info;
+    const { underlying, multiplier, contract_type, shortcode, purchase_time, date_expiry, tick_count } = contract_info;
     const { is_pathname_bot } = isBot();
+    const is_sold = !!contract_info.is_sold || is_contract_sold;
+    const is_turbos = isTurbosContract(contract_type);
+
+    const contract_type_list_info = [
+        {
+            is_param_displayed: multiplier,
+            displayed_param: `x${multiplier}`,
+        },
+        {
+            is_param_displayed: isTurbosContract(contract_type),
+            displayed_param: getContractSubtype(contract_type),
+        },
+    ];
+
+    const displayed_trade_param =
+        contract_type_list_info.find(contract_type_item_info => contract_type_item_info.is_param_displayed)
+            ?.displayed_param || '';
```

Comment:
@NikitK-deriv in your screenshot, you have null as a result because JS stops and returns the first falsy value it encounters when you use && operator. Your suggestion doesn't suit us unfortunately because we need the returned value to be of string type.